lib/vanagon/component/dsl.rb

```diff
@@ -184,7 +184,8 @@ def install_service(service_file, default_file = nil, service_name = @component.
           # Return here because there is no file to install, just a string read in
           return
         when "windows"
-          # Placeholder for Windows
+          @component.service = OpenStruct.new(:name => service_name, :service_file => service_file)
```

Comment:
So, in windows, it looks like the user, in order to install a service on windows for a component, will put
```
pkg.install_service "SourceDir\\ProgramFiles64Folder\\PuppetLabs\\Puppet\\bin\\ruby.exe"
```
into their component file.

Cool. That's all I was wondering.